website/docs/r/media_content_key_policy.html.markdown

```diff
@@ -125,6 +138,28 @@ The following arguments are supported:
 
 ---
 
+A `alternate_key` block supports the following:
+
+* `rsa_token_key_exponent` - (Optional) The RSA parameter exponent.
+
+* `rsa_token_key_modulus` - (Optional) The RSA parameter modulus.
+
+* `symmetric_token_key` - (Optional) The key value of the key. Specifies a symmetric key for token validation.
+
+* `x509_token_key_raw` - (Optional) The raw data field of a certificate in PKCS 12 format (X509Certificate2 in .NET). Specifies a certificate for token validation.
+
+-> **NOTE:** Each `alternate_key` block can only have one type of primary verification key: if you want use RSA you must provide `rsa_token_key_exponent` and `rsa_token_key_modulus`, if you want to use symmetric you need to provide `symmetric_token_key` and for x509 you must provide `x509_token_key_raw`. 
+
+---
+
+An `explicit_analog_television_output_restriction` block supports the following:
+
+* `best_effort` - (Optional) Indicates whether this restriction is enforced on a best effort basis. Possible values are `true` or `false`. Defaults to `false`. 
```

Comment:
this is a boolean so could we make this
```suggestion
* `best_effort_enabled` - (Optional) Indicates whether this restriction is enforced on a best effort basis. Possible values are `true` or `false`. Defaults to `false`. 
```
or
```suggestion
* `best_effort_enforced` - (Optional) Indicates whether this restriction is enforced on a best effort basis. Possible values are `true` or `false`. Defaults to `false`. 
```

Reply:
change to `best_effort_enforced`